### PR TITLE
minor edits + stage this for discussion

### PR DESCRIPTION
**Pull request by manzoid**

### CODE REVIEW

File: outlines/clean_code_1/cc1.code.ted.md [hunk 1]
```diff
@@ -1,13 +1,19 @@
 # TED
 
-Helping each other with code review is part of every day for programmers. You will take here the role of a code reviewer.
+Helping each other with code reviews is part of the everyday work of professional programming.
 
-Copy branch tasks and create new pull requests with your branch. Do not make changes to the code. Review code added by a branch. Add comments in places where you think code can be improved. Your comments should be helpful - indicating not only where but also how improvement can be made.
+Let's practice code reviewing.
+
+* Copy branch tasks and create new pull requests with your branch.
```
Comment (manzoid) [inline outlines/clean_code_1/cc1.code.ted.md:7]: I don't actually understand these instructions.

What's "copy branch tasks" mean?

File: outlines/clean_code_1/cc1.code.ted.md [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 
 # Things to be mindful about when checking the code
 
-1. Can you read a code out loud in plain English and understand it?
-1. Is naming conveying a clear intent?
-1. Is naming consistent across the project?
-1. Are names self-describing?
-1. Is formatting reflecting scopes?
\ No newline at end of file
+1. Can you read the code out loud in plain English and understand it?
```
Comment (manzoid) [inline outlines/clean_code_1/cc1.code.ted.md:15]: Not entirely sure what this means -- like, I'm translating the source code back into plain English?  Or what.

File: outlines/clean_code_1/cc1.code.ted.md [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
-Copy branch tasks and create new pull requests with your branch. Do not make changes to the code. Review code added by a branch. Add comments in places where you think code can be improved. Your comments should be helpful - indicating not only where but also how improvement can be made.
+Let's practice code reviewing.
+
+* Copy branch tasks and create new pull requests with your branch.
+* Do not make changes to the code.
+* Review code added by a branch.
+* Add comments in places where you think code can be improved.
+* Your comments should be actionable - indicating not only where but also how improvements can be made.
 
 # Things to be mindful about when checking the code
```
Comment (manzoid) [inline outlines/clean_code_1/cc1.code.ted.md:13]: Seems like we might want to factor out a common checklist / cheat sheet, and point to that?  Rather than listing things inline here.

**Comment by manzoid** [comment]
@trainings-au fyi. 

Also, why is your handle "trainings-au"?

File: outlines/clean_code_1/cc1.code.ted.md (hunk 1 above)
Comment (trainings-au) [reply outlines/clean_code_1/cc1.code.ted.md:7]: Oh yeah this was misunderstanding on my part how LMS works

File: outlines/clean_code_1/cc1.code.ted.md [hunk 4]
```diff
@@ -1,13 +1,19 @@
...
 
-Helping each other with code review is part of every day for programmers. You will take here the role of a code reviewer.
+Helping each other with code reviews is part of the everyday work of professional programming.
 
-Copy branch tasks and create new pull requests with your branch. Do not make changes to the code. Review code added by a branch. Add comments in places where you think code can be improved. Your comments should be helpful - indicating not only where but also how improvement can be made.
+Let's practice code reviewing.
+
+* Copy branch tasks and create new pull requests with your branch.
+* Do not make changes to the code.
+* Review code added by a branch.
```
[the remark this answers is not in the document]
Comment (trainings-au) [reply outlines/clean_code_1/cc1.code.ted.md:9]: I have rephrase it

File: outlines/clean_code_1/cc1.code.ted.md (hunk 2 above)
Comment (trainings-au) [reply outlines/clean_code_1/cc1.code.ted.md:15]: This is a reference to [this part](https://github.com/tpa-nextgen/modules-master/blob/master/outlines/clean_code_1/cc1.info.idea_of_clean_code.md#readable-code )

It might be also a non-native English speaker thing :D My observation is that people quite often don't see programming languages as languages. More like a set of abstract command-line instructions. So when they write code this way it becomes impossible to read it out loud. It is just gibberish like `var u = <UserLists>[]` that read as "variable u is list of type user lists" vs `var users = <User>[]` "variable users is list of type user". 

So when I would review junior code that is in that style, instead of going about why in this particular case their naming is confusing, I would just ask them to read the code out loud. Quite often I would get a reaction "oh I see" without the need for any further comments from my side.

Do you have an idea how we could rephrase it to convey the message better?

File: outlines/clean_code_1/cc1.code.ted.md (hunk 3 above)
Comment (trainings-au) [reply outlines/clean_code_1/cc1.code.ted.md:13]: hmm the problem I see with that is that list is not complete yet. It will be done after clean code II. Maybe we should link them in summative of cc II. So they can save it as bookmark for future reference?

**Comment by trainings-au** [comment]
> @trainings-au fyi.
> 
> Also, why is your handle "trainings-au"?

It is just account that was given to me when I joined the project. It got all necessary access to organisations.

File: outlines/clean_code_1/cc1.code.ted.md (hunk 2 above)
Comment (manzoid) [reply outlines/clean_code_1/cc1.code.ted.md:13]: yeah it's such an unusual practice that it's a bit hard to capture it in an introductory one-liner without it being confusing.  

It actually seems like a minor point to be honest, and I've never seen anyone actually do it.  Have you?  

We probably should stick with stuff we've actually seen and used ourselves? And not just tried it out but I mean it's a real part of our daily dev-work or team-work.

Comment (trainings-au) [reply outlines/clean_code_1/cc1.code.ted.md:13]: Oh yeah, this is from my personal experience with other programmers. 

At another company that I was working for they had trouble finding a mobile project and propose me to join the back-end for a while. So I also joined their messaging channel. Naturally quite often there were people consulting how to name stuff as back-end names often propagate to all platforms using it. After some time of me being there was a comment that I ended 80% of the naming discussion just by pointing to English and how code reads out.

But yeah it might be hard to convey that.
